sys/mips/include/frame.h

```diff
@@ -75,9 +75,12 @@
 typedef union {
 	uint32_t inst;
 	register_t pad; /* Ensure this is aligned the same way as other fields */
+
 } badinstr_t;
 
-/* Note: This must also match regnum.h and regdef.h */
+/* Use a incomplete type to avoid pointer-arithmetic */
```

Comment:
void * __capability would also avoid pointer arithmetic?

Reply:
GNU C treats it the same as `char * __ capability`...

Reply:
clang will warn for void*, so the -Werror build fails. Not using void* also has the benefit that implicit conversions won't happen. I feel like we should be explicit whenever we are modifying a program counter since it could be sealed.